Makefile.am

```diff
@@ -93,6 +93,7 @@ libmpd_a_SOURCES = \
 	src/command/MessageCommands.cxx src/command/MessageCommands.hxx \
 	src/command/ClientCommands.cxx src/command/ClientCommands.hxx \
 	src/command/PartitionCommands.cxx src/command/PartitionCommands.hxx \
+	src/external/common/MiscCommands.cxx src/external/common/MiscCommands.hxx \
```

Comment:
MPD already has Tidal and Qobuz support. Why this commit?

Reply:
Just for caryaudio.  We need login/logout by command, not in the mpd.conf. Some api changed at Qobuz and Tidal.